.env.example

```diff
@@ -0,0 +1 @@
+AST_FILE_PATH=path/to/ast.json
```

Comment:
- [ ] Wouldn't be better assuming`./ast.json`?

Reply:
The `ast.json` file is not supposed to be in this project root directory, nor should it be commited. Don't you think the path `./ast.json` would lead to this mistake?